src/redux/app/actions.ts

```diff
@@ -1,11 +1,18 @@
-// actions consts
-export const SET_ACCEPTED_TC_VERSION = 'SET_ACCEPTED_TC_VERSION';
+import { HintsComponent, HintsScreen } from './state';
 
-// actions creators
+export const SET_ACCEPTED_TC_VERSION = 'SET_ACCEPTED_TC_VERSION';
+export const SHOW_HINT = 'SHOW_HINT';
 
 export const appSetAcceptedTcVersion = (version: number) => {
     return {
         type: SET_ACCEPTED_TC_VERSION,
         data: version
     };
 };
+
+export const showHint = (screen: HintsScreen, component: HintsComponent) => {
```

Comment:
`showHint` is a little misleading in this case because when calling this action it doesn't show anything. Maybe `updateDisplayedHints`, i'm also ok if you find a better name :)